CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Haskell Package Versioning Policy](https://pvp.haskell.org/).
+
 ## [Unreleased]
 ### Added
 - custom-setup section
+  [#148](https://github.com/sol/hpack/pull/148)
+- Support `!include` directives
+  [#144](https://github.com/sol/hpack/pull/144)
+
+## [0.16.0] - 2017-01-11
+### Added
+- Warn when `name` is missing [#109](https://github.com/sol/hpack/issues/109)
+- Support globs in `c-sources`
```

Comment:
Basically we want this "Accepts glob patterns" comments, same to what we have for other fields